frontend/src/types.ts

```diff
@@ -0,0 +1,14 @@
+export interface Member {
+	id: string
+	first_name: string
+	last_name: string
+	email: string
+	member_type: number
+	faculty?: string
+	studentNumber?: number
+	watIAM?: string | null
+	mainInstrument: string
```

Comment:
Does this have to be a mandatory field?

Reply:
updated to be consistent with the other non-mandatory (uw student) fields